apps/core/scenes/profile/Profile.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import * as React from 'react';
+import { ScrollView } from 'react-native';
+import { StyleSheet, Button, Card } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { margaritaTokens } from '@kiwicom/margarita-design-tokens';
+import { Illustration, Text } from '@kiwicom/margarita-components';
+
+type Props = {||};
+
+export default class Profile extends React.Component<Props> {
+  handleSignIn = () => {
+    // @TODO
+  };
+
+  render() {
+    return (
+      <ScrollView
```

Comment:
What is reason for ScrollView not just a View?

Reply:
As Honza suggested it is replaced with `ContentContainer` component now. Which also uses `ScrollView` inside. And it is basically to have screen lets say future proof. For now there is not much content so scrolling can seems to be unnecessary. But if we for example expand description later and add more buttons it can easily go over the bottom edge.

But even with current content, scrolling can be necessary if user open the website in some really small window.